changelog/Microsoft.BitLocker.json

```
@@ -2,6 +2,56 @@
     "changelog": [
         {
             "ChangeList": [
+                {
+                    "Id": "94b01c30-2673-43c3-94a2-953c9270ca07",
+                    "ApiChange": "Method",
+                    "ChangedApiName": "bitlockerRecoveryKey-get",
+                    "ChangeType": "Addition",
+                    "Description": "Migrating [bitlockerRecoveryKey](/graph/api/bitlockerrecoverykey-get.md) method to v1.",
+                    "Target": "bitlockerRecoveryKey"
+                },
+                {
+                    "Id": "94b01c30-2673-43c3-94a2-953c9270ca07",
+                    "ApiChange": "Method",
+                    "ChangedApiName": "bitlocker-list-recoverykeys",
+                    "ChangeType": "Addition",
+                    "Description": "Migrating [bitlockerRecoveryKey](/graph/api/bitlocker-list-recoverykeys.md) method to v1.",
+                    "Target": "bitlockerRecoveryKey"
+                },
+                {
+                    "Id": "94b01c30-2673-43c3-94a2-953c9270ca07",
+                    "ApiChange": "Resource",
+                    "ChangedApiName": "bitlocker",
+                    "ChangeType": "Addition",
+                    "Description": "Migrating [bitlocker](/graph/resources/bitlocker.md) resource to v1.",
+                    "Target": "bitlocker"
+                },
+                {
+                    "Id": "94b01c30-2673-43c3-94a2-953c9270ca07",
+                    "ApiChange": "Resource",
+                    "ChangedApiName": "bitlockerrecoverykey",
+                    "ChangeType": "Addition",
+                    "Description": "Migrating [bitlockerRecoveryKey](/graph/resources/bitlockerrecoverykey.md) resource to v1.",
+                    "Target": "bitlockerRecoveryKey"
+                },
+                {
+                    "Id": "94b01c30-2673-43c3-94a2-953c9270ca07",
+                    "ApiChange": "Enum type",
+                    "ChangedApiName": "volumeType",
+                    "ChangeType": "Addition",
+                    "Description": "Added the **volumeType** enumeration type for migration to v1",
+                    "Target": "volumeType"
+                }
```

Applying the guidance
```suggestion
                {
                    "Id": "94b01c30-2673-43c3-94a2-953c9270ca07",
                    "ApiChange": "Enum type",
                    "ChangedApiName": "volumeType ",
                    "ChangeType": "Addition",
                    "Description": "Added the **volumeType** enumeration.",
                    "Target": "volumeType"
                }
```